### DISCUSSION TITLE
This is a test to see if discussions integrates with the Projects

### DISCUSSION THREAD

**Discussion by rachaelbradley** [w3c/silver#517] [General]
test

**Comment by bruce-usab** [comment]
Posting to confirm that I got this GitHub notification similar to what I get for Issues.

**Comment by jake-abma** [comment]
Test passed!

**Comment by rachaelbradley** [comment]
Thank you Jake and Bruce for letting me know. Disappointingly, I can't reference the discussion from the projects. Good test. Regretable result.

**Reply by ChrisLoiselle** [reply]
I attempted to file a GitHub issue with GitHub in the beta release of this but haven't heard back as of yet . Would be a great opportunity.
